fairseq/tasks/masked_lm.py

```diff
@@ -64,9 +65,7 @@ def __init__(self, args, dictionary):
 
     @classmethod
     def setup_task(cls, args, **kwargs):
-        paths = args.data.split(':')
-        assert len(paths) > 0
-        dictionary = Dictionary.load(os.path.join(paths[0], 'dict.txt'))
+        dictionary = Dictionary.load(str(Path(args.data) / 'dict.txt'))
```

Comment:
The issue is a bit different. We wanted to support multiple `data` arguments, since when training over large datasets it's often easier to shard the data and iterate over each shard during training.

We used `':'` as the separator, since that character isn't allowed in Unix paths. Unfortunately it breaks for Windows users. The correct solution is to use `os.pathsep` instead of hardcoding `':'`.